Fix the suggestion in error message when missing submodules

Reported by @torkar

If cloning with `git clone https://github.com/stan-dev/cmdstan` (without --recursive which doesnt clone stan and math) you get 

```
$ make -j16 build
ERROR: Missing Stan submodules.
Please run the following commands to fix:

git submodule init
git submodule update --recursive

And try building again
makefile:192: recipe for target 'build' failed
```
but this suggestion only clones the stan repo, not math.

Change the suggestion to
```
$ make -j16 build
ERROR: Missing Stan submodules.
Please run the following to fix:

git submodule update --init --recursive

And try building again
makefile:192: recipe for target 'build' failed
```

#### Current Version:
v2.23.0

--- a/makefile
+++ b/makefile
@@ -190,7 +190,7 @@ else
 .PHONY: build
 build:
 	@echo 'ERROR: Missing Stan submodules.'
-	@echo 'Please run the following commands to fix:'
+	@echo 'Please run the following to fix:'
 	@echo ''
 	@echo 'git submodule update --init --recursive'
 	@echo ''

--- a/makefile
+++ b/makefile
@@ -192,8 +192,7 @@ build:
 	@echo 'ERROR: Missing Stan submodules.'
 	@echo 'Please run the following commands to fix:'
 	@echo ''
-	@echo 'git submodule init'
-	@echo 'git submodule update --recursive'
+	@echo 'git submodule update --init --recursive'
 	@echo ''
 	@echo 'And try building again'
 	@exit 1

